File: src/DZarsky.TS3Viewer2.Web/src/api/models/ClientDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ClientDetailDto } from './ClientDetailDto';
import type { ClientType } from './ClientType';

export type ClientDto = {
    id?: number;
    channelId?: number;
    databaseId?: number;
    nickName?: string | null;
    type?: ClientType;
    detail?: ClientDetailDto;
};

File: src/DZarsky.TS3Viewer2.Domain/Server/Dto/ClientDto.cs
using TeamSpeak3QueryApi.Net.Specialized;

namespace DZarsky.TS3Viewer2.Domain.Server.Dto;

public sealed class ClientDto
{
    public int Id { get; set; }

    public int ChannelId { get; set; }

    public int DatabaseId { get; set; }

    public string? NickName { get; set; }

    public ClientType Type { get; set; }

    public ClientDetailDto? Detail { get; set; }
}
